client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,27 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
```

Comment:
Timezones should not be used in the password coffee, it's not a necessary data for this component because it's only password concerned.